frontend/app/src/utils/assets.ts

```diff
@@ -152,3 +152,17 @@ export const getSortItems = (
     });
   };
 };
+
+export const isEvmIdentifier = (identifier?: string) => {
+  if (!identifier) return false;
+  return identifier.startsWith('eip155');
+};
+
+export const getAddressFromEvmIdentifier = (identifier?: string) => {
+  if (!identifier) return '';
+  return identifier.split(':')[2] ?? '';
+};
+
+export const createEvmIdentifierFromAddress = (address: string) => {
+  return `eip155:1/erc20:${address}`;
```

Comment:
In the database we already have tokens from other chains. The `1` value should be configurable as it is the chain value and chain + address give you a unique identifier

Reply:
You should tag @lukicenturi here for this ^

Reply:
will be fixed by my incoming commit